Lib/test/test_zipfile.py

```diff
@@ -596,6 +597,20 @@ class LzmaTestsWithSourceFile(AbstractTestsWithSourceFile,
                               unittest.TestCase):
     compression = zipfile.ZIP_LZMA
 
+    def test_write_preset(self):
+        # Initialize an LZMA compressor with a preset and get the generated
+        # property header.
+        props_header = zipfile.LZMACompressor(preset=1).compress(b'')
+
+        # Write a ZIP archive with that LZMA compression preset and ensure
+        # that the property header above is written to that archive.
+        with io.BytesIO() as fp:
+            kwargs = {'compression': self.compression, 'compresslevel': 1}
```

Comment:
I don't think there's a need for this `kwargs` variable.

Reply:
Quite right. I put it there to avoid a long line.